pip/cmdoptions.py

```diff
@@ -498,6 +498,18 @@ def only_binary():
          "options to setup.py install. If you are using an option with a "
          "directory path, be sure to use absolute path.")
 
+download_options = partial(
```

Comment:
Oh, I see! My mistake, I had not noticed I failed to pass it down further! Do you think it makes sense to have this here? I wasn't sure, which is why `download_options` ended up being half-implemented I suppose. 

Since for `pip install` the `install_options` are only used in here https://github.com/pypa/pip/blob/develop/pip/commands/install.py#L307
I'm thinking maybe `download_options` doesn't need to exist the way `install_options` does. Shall I remove this?